src/modules/likes.js

```diff
@@ -0,0 +1,32 @@
+// Async function to get likes for a show
+export const getLikes = async () => {
+  await fetch(
+    'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/SzueLQl3FaQn1wJjDbsx/likes',
+  )
+    .then((response) => response.json())
+    // eslint-disable-next-line no-use-before-define
+    .then((data) => renderLikes(data));
+};
+const renderLikes = (likes) => {
+  const likesContainers = document.querySelectorAll('.likesContainer');
+  likesContainers.forEach((container) => {
+    // eslint-disable-next-line array-callback-return
```

Comment:
- [ ] In this case your filter is looking for item to return. You can put likes to return. Kindly remove the eslint-disabler